Cargo.toml

```diff
@@ -99,6 +99,7 @@ spl-associated-token-account = ">= 1.1.3, < 3.0"
 spl-concurrent-merkle-tree = "0.2.0"
 spl-noop = "0.2.0"
 spl-token = ">= 3.5.0, < 5.0"
+spl-token-2022 = "1.0"
```

Comment:
blockbuster is also using 1.0 but 2.x is available but requires solana-sdk ~1.18